mscore/mixer.cpp

```diff
@@ -249,8 +250,21 @@ void Mixer::showEvent(QShowEvent* e)
       QWidget::showEvent(e);
       activateWindow();
       setFocus();
+      getAction("toggle-mixer")->setChecked(true);
       }
 
+
+//---------------------------------------------------------
+//   hideEvent
+//---------------------------------------------------------
+
+void Mixer::hideEvent(QHideEvent* e)
+{
```

Comment:
6 space indent is needed too, I seem to have missed that in my last review.

Reply:
Sorry I missed that too. Fixed and pushed.